configs/env-example

```diff
@@ -27,14 +27,14 @@ E2E_TEST_STUDENT_PASSWORD=ADD_THE_PASSWORD_HERE_TO_RUN_LOCALLY
 E2E_TEST_BCeID_USERNAME=SIMS_E2ETests
 E2E_TEST_BCeID_PASSWORD=ADD_THE_PASSWORD_HERE_TO_RUN_LOCALLY
 
-# BCeID Web Service
+# BCeID Web Service on OC we do not the BCeID_ properties set
 BCeID_WEB_SERVICE_WSDL=https://gws1.test.bceid.ca/webservices/Client/V10/BCeIDService.asmx?wsdl
 BCeID_WEB_SERVICE_ONLINE_SERVICE_ID=
 BCeID_WEB_SERVICE_REQUESTER_USER_GUID=
 BCeID_WEB_SERVICE_AUTH_USER_NAME=
 BCeID_WEB_SERVICE_AUTH_USER_PASSWORD=
 
-# Devops/openshift Make api-init
+# Devops/openshift Make api-init - on  OC we only need API_ ones to run our application and not the BCeID_ ones
```

Comment:
To me "# Devops/openshift Make api-init" already says from where the variables belong to and it is clear that they belong to "Devops/openshift".
If you want to more clear, I don't think that "on OC" is making it clear.
Maybe you can add a comment mentioning that the first set of variables are used only to run the application locally.

Reply:
Done